backend/api/Areas/Property/Controllers/BuildingController.cs

```diff
@@ -56,6 +57,69 @@ public IActionResult GetBuilding(int id)
 
             return new JsonResult(building);
         }
+
```

Comment:
This functionality will be required when adding buildings with associated parcels. the new api will support associated parcels that already exist or are new. and will add/update as required.